datastores/meeting_datastore.ts

```diff
@@ -24,6 +25,10 @@ export const MeetingDatastore = DefineDatastore({
     agenda_trigger: {
       type: Schema.types.string,
     },
+    reminders: {
```

Comment:
Do we still need reminders on meetings? Or can we just save the meeting id on reminders if we need to associate them? If we're supporting updating reminders, we probably don't want to have to update the meeting.reminders in addition to the reminder itself.